code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -294,7 +294,7 @@ butcher
 
 /datum/aiTask/sequence/goalbased/build/drone/precondition()
 	var/mob/living/critter/flock/F = holder.owner
-	return F?.can_afford(20)
+	return F?.can_afford(20) && (F?.flock?.getComplexDroneCount() > 10) //prioritise egg laying in the early game
```

Comment:
So the flock values egg more than secure room that spessmen can't break into to lay egg in?

Reply:
yes, because the first 10 drones make it so much faster to convert and do flocky things.

Reply:
Also bear in mind you can tell a flockdrone to manually convert stuff or make flockbits to do it, but you can't tell a flockdrone to save up and lay an egg.